corehq/apps/data_dictionary/templates/data_dictionary/partials/valid_values_th_content.html

```diff
@@ -0,0 +1,7 @@
+{% load i18n %}
+{% trans "Valid Values &amp; Formats" %}
+<span
+  class="hq-help-template"
+  data-title="{% trans 'Valid values or format' %}"
```

Comment:
For html attributes, there's a `trans_html_attr` instead of `trans`, to handle quotes properly.